synapse/config/server.py

```diff
@@ -1168,6 +1181,22 @@ def generate_config_section(
         #
         #allow_per_room_profiles: false
 
+        # The largest allowed size for a user avatar. If not defined, no
+        # restriction will be imposed.
+        #
+        # Note that user avatar changes will not work if this is set without
+        # using Synapse's media repository.
```

Comment:
Should these settings be part of `synapse/config/repository.py` instead?

Reply:
Not sure, I see them more as profile-related settings since the action of uploading to the media repo won't be blocked on size and type.

Reply:
I don't feel strongly either way, just figured I'd ask!

Reply:
Fair enough :)